core/src/main/resources/hudson/model/DirectoryBrowserSupport/dir.jelly

```diff
@@ -70,6 +70,14 @@ THE SOFTWARE.
                     </j:forEach>
                   </td>
                   <j:if test="${!x.folder}">
+                    <td class="fileSize">
+                      <j:if test="${x.lastModifiedAsCalendar.timeInMillis &gt; 0}">
```

Comment:
Does this actually need to be an HTML entity?

Reply:
if you put ">" or "<" the jelly parser explode